signup.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="spacer100"></div>
+    <h1>Sign Up!</h1>
+    <div>
+        <form ng-submit="submit()" ng-controller="FormSignupCtrl">
+            Email:<br>
+            <input type="email" ng-model="email"><br>
+            Username:<br>
+            <input type="text" ng-model="username"><br>
+            Password:<br>
+            <input type="password" ng-model="password"><br>
+            <button ng-click="submit()">Submit</button>
```

Comment:
For these submit buttons you can style it with materialize (see [this link on button, especially the submit button with icon](http://materializecss.com/buttons.html))